cmd/eventkitd-sink-bigquery/main.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"net"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/jtolio/eventkit/pb"
+	"github.com/jtolio/eventkit/transport"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+type Application struct {
+	Name     string
+	Instance string
```

Comment:
the instance doesn't really feel like part of the application. it feels to me like it belongs more with the source address

Reply:
I guess all of this feels a bit like source information (source application, source instance, source version, source IP, etc)

Reply:
okay that's fair